app/templates/_webpack.config.js

```diff
@@ -3,31 +3,38 @@ import webpack from 'webpack';
 import UglifyJsPlugin from 'uglifyjs-webpack-plugin';
 import config from './<%= answers.projectName %>.Gulp/config';
 
+const production = process.env.NODE_ENV === 'production';
 const bundleFilename = '[name].bundle.js';
-const chunkFilename = '[name].[chunkhash].chunk.js';
-const vendorRegex = /node_modules/;
 
 export default {
-    mode: global.production ? 'production' : 'development',
+    mode: production ? 'production' : 'development',
     entry: {
         '<%= answers.projectName %>': `${config.js.src}/<%= answers.projectName %>/index.js`
     },
     output: {
         path: path.resolve(__dirname, config.js.dest),
         publicPath: `${config.js.dest.replace(config.baseDir, '')}/`,
         filename: bundleFilename,
-        chunkFilename: chunkFilename
+        chunkFilename: '[name].[chunkhash].chunk.js'
+    },
+    devtool: production ? 'source-map' : 'cheap-module-eval-source-map',
+    resolve: {
+        alias: {
+            '@': path.resolve(__dirname, config.js.src)
+        }
     },
-    devtool: global.production ? 'source-map' : 'cheap-module-eval-source-map',
     module: {
         rules: [
             {
                 test: /\.js$/,
-                exclude: vendorRegex,
                 loader: 'babel-loader',
                 options: {
-                    cacheDirectory: !global.production
-                }
+                    cacheDirectory: !production
+                },
+                include: [
+                    path.resolve(__dirname, config.js.src),
+                    path.resolve(__dirname, './node_modules/bootstrap')
```

Comment:
Why do we want to process Bootstrap with Babel? Can't we just use `import` stuff from the `dist` folder? Sorry, not a Bootstrap user, might have missed something 😅

Reply:
Hehe.. well, the dist folder is already compiled and bundled up. Imagine we only need the bootstrap modal in that case we wouldn't want the include the whole thing but just that. Also it makes it better for the treeshaking process. 💭

Reply:
But there are several `dist` folders out there, the one in `js` folder seems pretty reasonable to use. I'll take modal as an example, so we just import https://github.com/twbs/bootstrap/blob/v4-dev/js/dist/modal.js and you're good to go.

In any case I get your point 😄 My main concern was that if we apply this rule uniformly, then any new dependency involves subsequent change in `webpack.config.js` and that I'd like to avoid 😉

Reply:
Hey you know what 🤔 you are completely right 🦆🐇😀 . I just did a quick test with the `js/dist` folder and it didn't have any file size difference vs `js/src`. So, I have gone ahead and removed the line and if needed it can always be added by the developer incharge of a project.